sw/device/tests/alert_handler_lpg_clkoff_test.c

```diff
@@ -59,6 +59,8 @@ static dif_flash_ctrl_state_t flash_ctrl;
 
 static const uint32_t kPlicTarget = kTopEarlgreyPlicTargetIbex0;
 
+static volatile const uint32_t kTestIp = 0;
```

Comment:
```suggestion
static volatile const uint32_t kTestIp = 0;
static volatile const uint32_t kTestIp2 = 0;
```

Instead of using a single polling variable from SV, we can add a shadow copy to overcome the sw_backdoor mechanism's caching issue: 
- These two variables are updated by SV at the same time and to the same value (peri_idx+1). Here is the update code in SV: https://github.com/lowRISC/opentitan/pull/16408/files#r1025392431
- The C code polls both of them together. Please see the comment below too: https://github.com/lowRISC/opentitan/pull/16408/files#r1025383048

Reply:
We ought to avoid abusing the backdoor symbol overwrite mechanism like this haha :)! It was really only meant to be a one time thing that happens at the start of the test. We will worry about it in M3 though.